docs/openapi-fetch/index.md

```diff
@@ -10,6 +10,7 @@ openapi-fetch is a type-safe fetch client that pulls in your OpenAPI schema. Wei
 | :------------------------- | ---------: | :------------------------- |
 | openapi-fetch              |     `5 kB` | `300k` ops/s (fastest)     |
 | openapi-typescript-fetch   |     `4 kB` | `150k` ops/s (2× slower)   |
+| feature-fetch              |    `15 kB` | `300k` ops/s (not slower)  |
```

Comment:
😄 🚀 

You could also put `(fastest)` here as well if you wanted to.

Reply:
The performance difference between `openapi-fetch` and `feature-fetch` was negligible (0.0x) in my tests. I didn't update the statistics based on a single result set. Your updated benchmark in #1818 should provide more comprehensive data on relative performance 🚀